src/sentry/sentry_apps/external_requests/issue_link_requester.py

```diff
@@ -60,60 +67,79 @@ class IssueLinkRequester:
 
     def run(self) -> dict[str, Any]:
         response: dict[str, str] = {}
-
-        try:
-            request = send_and_save_sentry_app_request(
-                self._build_url(),
-                self.sentry_app,
-                self.install.organization_id,
-                f"external_issue.{ACTION_TO_PAST_TENSE[self.action]}",
-                headers=self._build_headers(),
-                method="POST",
-                data=self.body,
-            )
-            body = safe_urlread(request)
-            response = json.loads(body)
-        except (json.JSONDecodeError, TypeError):
-            raise SentryAppIntegratorError(
-                message=f"Unable to parse response from {self.sentry_app.slug}",
-                webhook_context={
-                    "error_type": "issue-link-requester.invalid-json",
-                    "uri": self.uri,
-                    "response": body,
-                    "installation_uuid": self.install.uuid,
-                },
-                status_code=500,
-            )
-        except RequestException as e:
-            error_type = "issue-link-requester.error"
-            extras = {
-                "sentry_app": self.sentry_app.slug,
-                "installation_uuid": self.install.uuid,
-                "project": self.group.project.slug,
-                "group": self.group.id,
+        event = SentryAppEventType(f"external_issue.{ACTION_TO_PAST_TENSE[self.action]}")
```

Comment:
nit: should `self.action` be typed as a `StrEnum` that is either `create` or `link` so it's not possible for this to fail?